pkg/kubelet/kubelet.go

```diff
@@ -1487,6 +1503,22 @@ func getPhase(spec *api.PodSpec, info api.PodInfo) api.PodPhase {
 	}
 }
 
+func isPodReady(spec *api.PodSpec, info api.PodInfo) bool {
```

Comment:
Again, a comment about the behavior would be useful: pod is not ready if any container is not ready.